src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -193,40 +183,9 @@ export const QueryResultPane = () => {
 
         return () => observer.disconnect();
     }, [metadata?.resultSetSummaries]);
-    const [columns] =
-        useState<TableColumnDefinition<qr.IMessage>[]>(columnsDef);
-    const items = metadata?.messages ?? [];
-
-    const sizingOptions: TableColumnSizingOptions = {
-        time: {
-            minWidth: 100,
-            idealWidth: 100,
-            defaultWidth: 100,
-        },
-        message: {
-            minWidth: 500,
-            idealWidth: 500,
-            defaultWidth: 500,
-        },
-    };
-
-    const [columnSizingOption] =
-        useState<TableColumnSizingOptions>(sizingOptions);
-    const { getRows, columnSizing_unstable, tableRef } = useTableFeatures(
-        {
-            columns,
-            items: items,
-        },
-        [
-            useTableColumnSizing_unstable({
-                columnSizingOptions: columnSizingOption,
-            }),
-        ],
-    );
-    const rows = getRows();
 
+    // ========================= RESULT GRID =========================
```

Comment:
FYI: you can do this if you want vscode to recognize this as a region for code collapsing

```
//#region Result Grid
stuff here
//#endregion
```

Reply:
updated